style1.css

```diff
@@ -390,3 +390,285 @@ textarea::placeholder {
 .cont {
   margin: 20px 0;
 }
+
+@media only screen and (min-width: 768px) {
+  #navigation {
+    display: flex;
+    justify-content: space-between;
+    padding: 16px 66px;
+    box-shadow: 0 1px 10px -5px grey;
+    position: fixed;
+    height: 72px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .menu-bar {
+    display: none;
+  }
+
+  .nav-list {
+    margin-left: initial;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 10px;
+    list-style: none;
+    text-transform: capitalize;
+  }
+
+  .nav-list .nav-content a {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .nav-list .nav-content a:hover {
+    text-decoration: underline;
+    color: #6070ff;
+    transition: 0.1s;
+    font-weight: bold;
+  }
+
+  .hero-section {
+    background-image: url(/images1/Header-bg.svg);
+    background-size: 100% 691.46px;
+    background-position-x: right;
+    padding-top: 8rem;
+    padding-bottom: 9rem;
+    height: 846px;
+  }
+
+  .hero-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    width: 70%;
+    margin: auto;
+    margin-top: 100px;
+    gap: 20px;
+  }
+
+  #name {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    margin: 20px 0;
+    height: 50px;
+  }
+
+  .details {
+    width: 64%;
+  }
+
+  #work-section-mobile {
+    display: none;
+  }
+
+  #work-section-desktop {
+    width: 80%;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-flow: row;
+    justify-items: center;
+    align-items: center;
+    margin-top: 46px;
+  }
+
+  .project {
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 36px;
+    max-width: initial;
+  }
+
+  .project:hover {
+    cursor: pointer;
+    border: 2px solid #a7aeff;
+  }
+
+  .work-img {
+    display: grid;
+    place-items: center;
+    width: 100%;
+  }
+
+  .work-img img {
+    width: 100%;
+  }
+
+  .work-tools {
+    width: 108%;
+  }
```

Comment:
- [ ] Kindly increase the padding for each list item and align the text vertically and horizontally center:
> your design: 
![image](https://user-images.githubusercontent.com/91486791/236467181-c7479d59-d4ec-427a-9d0d-a426fe5dd78b.png)
> figma: 
![image](https://user-images.githubusercontent.com/91486791/236467316-8ee1dcd1-6a9c-4805-93d0-6266e4e4d76c.png)

Reply:
- [ ] Corrected :white_check_mark: